src/developer-testing/developer-testing.service.ts

```diff
@@ -0,0 +1,172 @@
+
+    // Now create a CDD claim for each Identity
+    const createCddCalls = identities.map(({ did }) =>
+      identity.addClaim(did, { CustomerDueDiligence: did }, null)
+    );
+
+    // and provide POLYX for those that are supposed to get some
+    const initialPolyxCalls = accounts
+      .filter(({ initialPolyx }) => initialPolyx.gt(0))
+      .map(({ address, initialPolyx }) =>
+        balances.transfer(address, initialPolyx.toNumber() * unitsPerPolyx)
+      );
+
+    await this.polymeshService.execTransaction(signerAddress, utility.batchAtomic, [
+      ...createCddCalls,
+      ...initialPolyxCalls,
+    ]);
+
+    return identities;
+  }
+
+  /**
+   * @note relies on `//Alice` having `sudo` permission
+   */
+  public async batchCreateCddProviders(identities: Identity[]): Promise<void> {
+    const {
+      polymeshService: {
+        polymeshApi: {
+          _polkadotApi: {
+            tx: { cddServiceProviders, sudo, utility },
+          },
+        },
+      },
+      alicePair,
+    } = this;
+    const cddCalls: SubmittableExtrinsic<'promise'>[] = [];
+
+    identities.forEach(({ did }) => {
+      const addCddProviderTx = cddServiceProviders.addMember(did);
+      cddCalls.push(addCddProviderTx);
+    });
+
+    const batchTx = utility.batchAtomic(cddCalls);
+
+    await this.polymeshService.execTransaction(alicePair, sudo.sudo, batchTx);
+  }
+
+  /**
+   * @note relies on `//Alice` having `sudo` permission
+   */
+  public async batchCreateAdmins({ addresses }: CreateCddProviders): Promise<Identity[]> {
+    const identities = await this.batchMockCddClaim(addresses);
+
+    await this.batchCreateCddProviders(identities);
+
+    return identities;
+  }
+
+  /**
+   * @note relies on `//Alice` having `sudo` permission
+   */
+  public async batchMockCddClaim(addresses: string[]): Promise<Identity[]> {
+    const {
+      polymeshService: {
+        polymeshApi: {
+          _polkadotApi: {
+            tx: { testUtils, utility, balances, sudo },
+          },
+        },
+      },
+      alicePair,
+    } = this;
+
+    const cddCalls = addresses.map(address => testUtils.mockCddRegisterDid(address));
+    const balanceCalls = addresses.map(address =>
+      balances.setBalance(address, initialAdminPolyx, 0)
+    );
+    const balanceTx = sudo.sudo(utility.batchAtomic(balanceCalls));
+
+    await this.polymeshService.execTransaction(alicePair, utility.batchAtomic, [
+      ...cddCalls,
+      balanceTx,
+    ]);
+
+    await this.polymeshService.execTransaction(alicePair, sudo.sudo, balanceTx);
+
+    const accounts = await this.fetchAccountsForAddresses(addresses);
+
+    return this.fetchAccountsIdentities(accounts);
+  }
+
+  private get alicePair(): KeyringPair {
+    if (!this._alicePair) {
+      const ss58Format = this.polymeshService.polymeshApi.network.getSs58Format().toNumber();
+      const keyring = new Keyring({ type: 'sr25519', ss58Format });
+      this._alicePair = keyring.addFromUri('//Alice');
```

Comment:
IMO we should add an extra variable in environments and default it to '//Alice'. That way if anyone wants to use this setup differently they don't have to edit the files